modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -201,7 +201,7 @@ component accessors="true" singleton threadSafe{
 	function siteBaseURL(){
 		return replacenocase( 
 			getRequestContext().buildLink( linkTo='', ssl=getRequestContext().isSSL() ), 
-			"index.cfm", 
+			"index.cfm/", 

```

Comment:
I have checked all of our themes, its only being used like this `<base href="#cb.siteBaseURL()#" />`

Other than my sitemap code, there is nothing else using this.
I think its safe to change.